ui/pages/confirm-approve/confirm-approve.js

```diff
@@ -163,101 +176,106 @@ export default function ConfirmApprove() {
+              showEditApprovalPermissionModal={({
+                /* eslint-disable no-shadow */
+                customTokenAmount,
+                decimals,
+                origin,
+                setCustomAmount,
+                tokenAmount,
+                tokenBalance,
+                tokenSymbol,
+                /* eslint-enable no-shadow */
+              }) =>
+                dispatch(
+                  showModal({
+                    name: 'EDIT_APPROVAL_PERMISSION',
+                    customTokenAmount,
+                    decimals,
+                    origin,
+                    setCustomAmount,
+                    tokenAmount,
+                    tokenBalance,
+                    tokenSymbol,
+                  }),
+                )
+              }
+              data={customData || data}
+              toAddress={toAddress}
+              currentCurrency={currentCurrency}
+              nativeCurrency={nativeCurrency}
+              ethTransactionTotal={ethTransactionTotal}
+              fiatTransactionTotal={fiatTransactionTotal}
+              hexTransactionTotal={hexTransactionTotal}
+              useNonceField={useNonceField}
+              nextNonce={nextNonce}
+              customNonceValue={customNonceValue}
+              updateCustomNonce={(value) => {
+                dispatch(updateCustomNonce(value));
+              }}
+              getNextNonce={() => dispatch(getNextNonce())}
+              showCustomizeNonceModal={({
+                /* eslint-disable no-shadow */
+                useNonceField,
+                nextNonce,
+                customNonceValue,
+                updateCustomNonce,
+                getNextNonce,
+                /* eslint-disable no-shadow */
+              }) =>
+                dispatch(
+                  showModal({
+                    name: 'CUSTOMIZE_NONCE',
+                    useNonceField,
+                    nextNonce,
+                    customNonceValue,
+                    updateCustomNonce,
+                    getNextNonce,
+                  }),
+                )
+              }
+              warning={submitWarning}
+              txData={transaction}
+              fromAddressIsLedger={fromAddressIsLedger}
+              chainId={chainId}
+              rpcPrefs={rpcPrefs}
+              isContract={isContract}
+              isMultiLayerFeeNetwork={isMultiLayerFeeNetwork}
+              supportsEIP1559V2={supportsEIP1559V2}
             />
-          )}
-        </>
-      }
-      hideSenderToRecipient
-      customTxParamsData={customData}
-    />
+            {showCustomizeGasPopover && !supportsEIP1559V2 && (
+              <EditGasPopover
+                onClose={closeCustomizeGasPopover}
+                mode={EDIT_GAS_MODES.MODIFY_IN_PLACE}
+                transaction={transaction}
+              />
+            )}
+            <EditGasFeePopover />
+            <AdvancedGasFeePopover />
```

Comment:
Should we wrap these like :
```
{showCustomizeGasPopover && supportsEIP1559V2 && (
             <EditGasFeePopover />
            <AdvancedGasFeePopover />
            )}
```

Reply:
`EditGasFeePopover` and `AdvancedGasFeePopover` will return null if `supportsEIP1559V2` is false.